src/components/zoom-to-marker.js

```diff
@@ -0,0 +1,14 @@
+import { useMap } from "react-leaflet"
+
+export default function PanToMarker(position) {
+    const map = useMap()
+    const currentLocation = map.getCenter()
+    const location = position.position ? position.position : [51.505, -0.09]
+
+        map.flyTo( location,10 );
+        setTimeout(() => {
+            map.flyTo( [51.505, -0.09],3 )
```

Comment:
I see these 2 numbers are used twice, maybe they could be moved to a constant for reuse?

Reply:
Ack! That was my test data.  I should have stopped playing wiht this earlier. Good catch!